src/main/java/teammates/common/util/StringHelper.java

```diff
@@ -180,19 +180,20 @@ public static String toDecimalFormatString(double doubleVal) {
     }
 
     public static String toUtcFormat(double hourOffsetTimeZone) {
-        String utcFormatTimeZone = "UTC";
+        StringBuilder utcFormatTimeZone = new StringBuilder();
+        utcFormatTimeZone.append("UTC");
         if (hourOffsetTimeZone != 0) {
             if ((int) hourOffsetTimeZone == hourOffsetTimeZone) {
-                utcFormatTimeZone += String.format(" %+03d:00", (int) hourOffsetTimeZone);
+                utcFormatTimeZone.append(String.format(" %+03d:00", (int) hourOffsetTimeZone));
             } else {
-                utcFormatTimeZone += String.format(
+                utcFormatTimeZone.append(String.format(
                                             " %+03d:%02d",
                                             (int) hourOffsetTimeZone,
-                                            (int) (Math.abs(hourOffsetTimeZone - (int) hourOffsetTimeZone) * 300 / 5));
+                                            (int) (Math.abs(hourOffsetTimeZone - (int) hourOffsetTimeZone) * 300 / 5)));
             }
         }
 
-        return utcFormatTimeZone;
+        return utcFormatTimeZone.toString();
```

Comment:
I think here's another place where the StringBuilder is not needed. In the conditionals, just return the original `utcFormatTimeZone` appended with the appropriate additions.